host/xtest/regression_6000.c

```diff
@@ -2254,3 +2254,62 @@ static void xtest_tee_test_6020_single(ADBG_Case_t *c, uint32_t storage_id)
 DEFINE_TEST_MULTIPLE_STORAGE_IDS(xtest_tee_test_6020)
 ADBG_CASE_DEFINE(regression, 6020, xtest_tee_test_6020,
 		 "Object IDs in SHM (negative)");
+
+static void xtest_tee_test_6021_single(ADBG_Case_t *c, uint32_t storage_id)
+{
+	const uint32_t flags = TEE_DATA_FLAG_ACCESS_READ |
+			       TEE_DATA_FLAG_ACCESS_WRITE |
+			       TEE_DATA_FLAG_ACCESS_WRITE_META;
```

Comment:
`TEE_DATA_FLAG_OVERWRITE` to ensure object is destroyed and recreated?

Reply:
OK, I'll add it to the create call.